docs/architecture/protocol-changes.md

```diff
@@ -36,5 +36,157 @@ sequenceDiagram
+    participant StorageProvider2 as StoregeProvider [l-z]
+
+    Note right of ocDAV: translate ownCloud flavoured webdav<br>into CS3 API requests
+        ocDAV->>+Gateway: ListContainer({spaceid}, path: {relative/path})
+    Note right of Gateway: find address of the storage provider<br>that is responsible for the space
+        Gateway->>+StorageRegistry: ListStorageProviders({spaceid})
+        StorageRegistry-->>-Gateway: []ProviderInfo
+    Note right of Gateway: forward request to<br>correct storage provider
+    alt username starting with a-k
+        Gateway->>+StorageProvider1: ListContainer({spaceid}, path: {relative/path})
+        StorageProvider1-->>-Gateway: []ResourceInfo
+    else username starting with l-z
+        Gateway->>+StorageProvider2: ListContainer({spaceid}, path: {relative/path})
+        StorageProvider2-->>-Gateway: []ResourceInfo
+    end 
+        Gateway-->>-ocDAV: []ResourceInfo
+{{</mermaid>}}
+
+
+### Old /dav/files/{username} endpoint with username and a path relative to the users home
+
+To route a PROPFIND request against the old webdav endpoints ocdav first has to build a CS3 namespace prefix, eg. `/users/{{.Id.OpaqueId}}` to the users home. 
+
+{{<mermaid class="text-center">}}
+%%{init: {"sequence": { "showSequenceNumbers":true, "messageFontFamily":"courier", "messageFontWeight":"normal", "messageFontSize":"11"}}}%%
+%% font weight is a css bug: https://github.com/mermaid-js/mermaid/issues/1976
+%% edit this diagram by pasting it into eg. https://mermaid.live
+sequenceDiagram
+    participant Client
+    participant ocDAV
+    participant Gateway
+
+    opt old /dav/files/{username} endpoint with username and a path relative to the users home
+    Note right of Client: translate ownCloud flavoured webdav<br>into CS3 API requests
+        Client->>+ocDAV: PROPFIND /dav/files/{username}/{relative/path}
+    Note right of ocDAV: translate ownCloud flavoured webdav<br>into CS3 API requests
+        ocDAV->>+Gateway: GetUser({username})
+        Gateway-->>-ocDAV: User
+    Note right of ocDAV: build path prefix to user home
+        ocDAV->>+ocDAV: {namespace/prefix} = ApplyLayout({path layout}, User), eg. /users/e/einstein
+    Note right of ocDAV: look up the space responsible for a path
+        ocDAV->>+Gateway: ListStorageSpaces(path: {namespace/prefix}/{relative/path})
+        Gateway-->>-ocDAV: []StorageSpace 
+    Note right of ocDAV: make actual request with space and relative path
+        ocDAV->>+Gateway: ListContainer({spaceid}, path: {relative/path})
+        Gateway-->>-ocDAV: []ResourceInfo 
+        ocDAV-->>-Client: 207 Multistatus 
+    end
+{{</mermaid>}}
+
+Why does ocis use a path based lookup instead of looking up the current users home using the user id and a space type filter? There are deployments that use a global namespace at the legacy `/webdav` endpoint. To support that use case the gateway allows looking up spaces using their mount path.
```

Comment:
```suggestion
**Handling legacy global namespace webdav endpoints**

Why does ocis use a path based lookup instead of looking up the current users home using the user id and a space type filter? There are deployments that use a global namespace at the legacy `/webdav` endpoint. To support that use case, the gateway allows looking up spaces using their mount path.
```